docs/install/sm.md

```diff
@@ -1 +1,94 @@
-Two sections (SM on CF and SM on K8S)
\ No newline at end of file
+# Run Service Manager
```

Comment:
TODO: add dep ensures / git clones

Mention to not use go get

Reply:
Bump - add these

Reply:
> Bump - add these

What should be added?

Reply:
Dep ensures/ git clones - I can see you've already added those :)